example/nce-loss/toy_nce.py

```diff
@@ -56,16 +60,16 @@ def provide_label(self):
 
 
 class DataIter(mx.io.DataIter):
-    def __init__(self, count, batch_size, vocab_size, num_label, feature_size):
+    def __init__(self, count, batch_size_, vocab_size_, num_label_, feature_size_):
         super(DataIter, self).__init__()
-        self.batch_size = batch_size
+        self.batch_size = batch_size_
```

Comment:
Does trailing underscore follow google's style guide for python?
https://google.github.io/styleguide/pyguide.html#Naming

Reply:
Trailing underscores for name clashes are part of PEP8: https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments
The reason for the change is that variable names in the function clash with variable names in the outer scope.